brave/build.gradle

```diff
@@ -1,5 +1,6 @@
 dependencies {
     compile 'io.zipkin.brave:brave'
+    compile 'io.zipkin.brave:brave-instrumentation-http'
```

Comment:
there is also http-tests which includes the IT which can be used to validate behavior